Stripe/STPPaymentOptionTableViewCell.m

```diff
@@ -117,6 +122,27 @@ - (void)configureWithPaymentOption:(id<STPPaymentOption>)paymentOption theme:(ST
     [self setNeedsLayout];
 }
 
+- (void)configureForFPXRowWithTheme:(STPTheme *)theme {
+    self.paymentOption = nil;
+    self.theme = theme;
+
+    self.backgroundColor = theme.secondaryBackgroundColor;
+    
+    // Left icon
+    self.leftIcon.image = [STPImageLibrary bankIcon];
+    self.leftIcon.tintColor = [self primaryColorForPaymentOptionWithSelected:NO];
+
+    // Title label
+    self.titleLabel.font = theme.font;
+    self.titleLabel.textColor = self.theme.primaryForegroundColor;
+    self.titleLabel.text = STPLocalizedString(@"Online Banking (FPX)", @"Button to pay with a Bank Account (using FPX).");
+
+    // Checkmark icon
+    self.checkmarkIcon.hidden = YES;
```

Comment:
does this get reset if the cell is reused for a non-FPX option?

Reply:
It does! For selectable cells, the hidden field is populated with the selected state of the card.